examples/demo-rollup/src/main.rs

```diff
@@ -12,6 +12,8 @@ use demo_stf::runner_config::from_toml_path;
 use demo_stf::runtime::{get_rpc_methods, GenesisConfig};
 use jsonrpsee::core::server::rpc_module::Methods;
 use jupiter::da_service::CelestiaService;
+#[cfg(feature = "experimental")]
+use jupiter::da_service::DaServiceConfig;
```

Comment:
Why this config is experimental?

Reply:
It is not, I just use the type in experimental code and need to guard it with the flag so we don't get an "unused import" warning.